docs/tutorials/getting-started.md

```diff
@@ -42,7 +42,47 @@ with the tests.
 
 ## Run the test suite
 
-🚧 **TODO**
+#### Redash Setup
+The Redash instance must be setup before running the tests. Start the images ```docker-compose up -d```. This will download the images if you don't have them.
+
+Run these following commands to setup Redash:
+1. ```docker-compose run --rm server create_db```
+2. ```docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"```
+3. ```docker-compose run --rm server /app/manage.py users create_root root@example.com "rootuser" --password "IAMROOT" --org default```
+4. ```docker-compose run --rm server /app/manage.py ds new "ui-tests" --type "url" --options '{"title": "uitests"}'```
```

Comment:
Can you explain what these commands do?

For example:

Create the database:
```
docker-compose run --rm server create_db
```

Create the database tables:

```text
docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"
```

...

This may seem obvious but it will help newcomers understand what they are doing when running these commands.